http/src/main/resources/reference.conf

```diff
@@ -67,7 +67,7 @@ dataproc {
     region = "us-central1"
   }
 
-  customDataprocImage = "projects/broad-dsp-gcr-public/global/images/leo-0322-2-image-dataproc-2-0-27-debian10-52fca9d"
+  customDataprocImage = "projects/broad-dsp-gcr-public/global/images/custom-leo-image-dataproc-2-0-27-debian10-aa373a8"
```

Comment:
can you rebase properly to get rid of non related commits

Reply:
Done.